### PR TITLE
Separate login view

### PR DESCRIPTION
**Pull request by HaiSSer666**
created a new login form

### CODE REVIEW

File: NewsAggregator/Classes/LoginManager.cs [hunk 1]
```diff
@@ -34,6 +36,48 @@ public void LoginWithPIN()
...
+        {
+            this.authenticationContext = AuthFlow.InitAuthentication(credentials);
+            Console.WriteLine();
+            try
+            {
+                System.Diagnostics.Process.Start(authenticationContext.AuthorizationURL);
+            }
+            catch (System.NullReferenceException ex)
+            {
+                MessageBox.Show("Error is occured. Authentication was not successful.\n Try to start app one more time.");
```
Comment (MaxRandom) [inline NewsAggregator/Classes/LoginManager.cs]: UI code should not be a part of Login Manager

File: NewsAggregator/Classes/Serealizator.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+            }
+            catch (Exception ex)
+            {
+                return null;
+            }
+        }
+    }
+}
+
+// c:\users\admin\documents\visual studio 2015\Projects\NewsAggregator\
```
Comment (MaxRandom) [inline NewsAggregator/Classes/Serealizator.cs]: Remove this comment

File: NewsAggregator/Program.cs [hunk 3]
```diff
@@ -16,7 +17,9 @@ static void Main()
         {
             Application.EnableVisualStyles();
             Application.SetCompatibleTextRenderingDefault(false);
-            Application.Run(new MainForm());
+
+            LoginManager loginManager = LoginManager.Instance;
+            loginManager.InitialiseSession();
```
Comment (MaxRandom) [inline NewsAggregator/Program.cs]: `InitialiseSession` -> `InitializeSession`

**Review by MaxRandom — CHANGES_REQUESTED** [review]

**Comment by HaiSSer666** [comment]
change code according to @MaxRandom comments

**Review by MaxRandom — APPROVED** [review]

**Pull request merged**